### PR TITLE
Korotkevich/task2

### PR DESCRIPTION
**Pull request by GrishaShreider**
Made Task2

### CODE REVIEW

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+'''Реализовать:
+1.текстовый вывод сегодняшнего времени
+2.текстовый вывод времени, ведомого с консоли (чч:мм).
+(дать возможность выбора режима работы программы)
+
+Для получения свободного времени можно использовать модуль datetime.
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Korotkevich_Grigory/Task_2/Task2.py:7]: google translate and back? very strange wording, not from my task description

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+14: ['четырнадцать', 'четарнадцати'], 15: ['пятнадцать', 'пятнадцати'],
+16: ['шестнадцать'], 17: ['семнадцать'], 18: ['восемнадцать'], 19: ['девятнадцать'],
+20: ['двадцать'], 30: ['тридцать'], 40: ['сорок'], 50: ['пятьдесят']
+}
+
+counting_hours = {
+0: ['ноль'],1: ['час', 'первого'],2: ['два', 'второго'],3: ['три', 'третьего'],4: ['четыре', 'четвертого'],5: ['пять', 'пятого'],
+6: ['шесть', 'шестого'],7: ['семь', 'седьмого'],8: ['восемь', 'восьмого'],9: ['девять', 'девятого'],10: ['десять', 'десятого'],
+11: ['одинадцать', 'одинадцатого'],12: ['двенадцать', 'двенадцатого'],
+}
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Korotkevich_Grigory/Task_2/Task2.py:39]: a decent data structure

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+player = int(input('Enter 1 if you want to know the current time!\n'
+                   'Enter 2 and you will need to manually enter the time in hh:mm format!\n'))
+
+
+if player == 1:
+    time = dt.datetime.now().time()
+    time = time.strftime('%H:%M')
+
+elif player == 2:
+    time = input()
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Korotkevich_Grigory/Task_2/Task2.py:51]: so first of all, show user more explicitly that you want they to input data

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    time = time.replace(' ', '')
+
+time = time.split(':')
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Korotkevich_Grigory/Task_2/Task2.py:54]: secondly, if you specify some format for user to comply, make sure that user entered some valid data (check at least for length and ':' sign) before splitting

**Review by MikitaTsiarentsyeu — APPROVED** [review]
A very good solution, you found a right balance between an a dictionary structure and a solution logic, good for you. The score is 9.5

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py (hunk 1 above)
Comment (GrishaShreider) [reply Tasks/Korotkevich_Grigory/Task_2/Task2.py:7]: Yes, it is he who is the culprit of the distortion :)

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py (hunk 3 above)
Comment (GrishaShreider) [reply Tasks/Korotkevich_Grigory/Task_2/Task2.py:51]: ok, I'll take it into account

File: Tasks/Korotkevich_Grigory/Task_2/Task2.py (hunk 4 above)
Comment (GrishaShreider) [reply Tasks/Korotkevich_Grigory/Task_2/Task2.py:54]: ok, I'll take it into account

**Pull request merged**